### PR TITLE
get object by object url

### PR DESCRIPTION
**Pull request by Niek-Ritense**

### CODE REVIEW

File: zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ObjectResource.kt [hunk 1]
```diff
@@ -53,4 +54,10 @@ class ObjectResource(
     ): ResponseEntity<URI> {
         return ResponseEntity.ok(zaakObjectService.patchObjectFromManagementId(objectManagementId, objectId, jsonNode))
     }
+
+    @GetMapping
```
Comment (Klaas-Ritense) [inline zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ObjectResource.kt:58]: Missing HttpSecurityConfigurer

Comment (Klaas-Ritense) [reply zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ObjectResource.kt:58]: Also, why do we need this?

Comment (Niek-Ritense) [reply zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ObjectResource.kt:58]: The Adam implementation needs this to get a call from form.io to the valtimo backend

**Review by Klaas-Ritense — APPROVED** [review]

File: zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ObjectResource.kt (hunk 1 above)
Comment (Klaas-Ritense) [reply zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ObjectResource.kt:58]: Not sure what you mean here. This endpoint here has nothing to do with form.io

Comment (Niek-Ritense) [reply zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ObjectResource.kt:58]: i was asked to make this endpoint by the implementation team to help them with a need that they would be able to use this call directly from within a form.io form or bpmn process

**Pull request merged**